src/components/ReportCard.js

```diff
@@ -3,64 +3,88 @@ import styled from 'styled-components';
 import { brandColours, brandFonts, minBreakpointQuery } from '../styles';
 
 //Initialised for transform hover animation
-const StyledContent = styled.div`
-  position: absolute;
-  left: 0px;
-  bottom: 0px;
-  max-width: 280px;
+const StyledLink = styled.a`
+  font-family: ${brandFonts.sans};
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 20px;
+  text-transform: uppercase;
+  color: ${brandColours.quinary};
+  transition: transform 0.4s ease-in-out;
+  transform: translateY(80px);
+
+  &:hover {
+    text-decoration: underline;
+  }
+`;
+
+const StyledHeader = styled.h3`
+  font-family: ${brandFonts.serif};
+  font-size: 25px;
+  font-weight: 700;
+  color: ${brandColours.tertiary};
+  line-height: 35px;
+  justify-self: start;
+  margin-bottom: 20px;
   transition: transform 0.4s ease-in-out;
-  transform: translateY(38%);
-  margin-left: 25px;
-  margin-bottom: 35px;
+  transform: translateY(80px);
 
   ${minBreakpointQuery.medium`
-    left: 0px;
-    bottom: 0px;
-    margin-left: 30px;
-    margin-right: 30px;
-    margin-bottom: 20px;
+    font-size: 20px;
   `}
 
   ${minBreakpointQuery.large`
-    margin-left: 45px;
-    margin-right: 45px;
-    margin-bottom: 40px;
+    font-size: 30px;
+    margin-bottom: 50px;
   `}
 `;
 
-//Initialised for gradient hover effect
-const StyledOverlay = styled.div`
-  position: absolute;
-  height: 100%;
-  width: 100%;
-  background: linear-gradient(rgba(0, 0, 0, 0.01) 0%, rgba(14, 50, 41, 1) 100%);
-`;
-
-const StyledReportCard = styled.div`
-  overflow: hidden;
+const StyledReportCard = styled.article`
   position: relative;
+  display: grid;
   cursor: pointer;
   border-radius: 15px;
-  margin-bottom: 30px;
+  max-width: 380px;
 
   ${minBreakpointQuery.medium`
-    margin-left: 10px;
-    margin-right: 10px;
+    max-height: 470px;
   `}
 
-  &:hover ${StyledContent} {
-    transform: translateY(0%);
+  &:hover ${StyledLink} {
+    transform: translateY(0px);
+  }
+
+  &:hover ${StyledHeader} {
+    transform: translateY(0px);
+  }
+
+  &:after {
```

Comment:
This is fine, but in this case since the StyledContentOuter is covering the whole area the gradient can go on there, the gradient doesn't need to change on hover either so a background on StyledContentOuter is fine. Then you can remove the z-index as well from StyledContentOuter as won't be needed.